documents/purchaseorder/handler.go

```diff
@@ -25,12 +20,11 @@ var apiLog = logging.Logger("purchaseorder-api")
 // anchoring, sending, finding stored purchase order document
 type grpcHandler struct {
 	service Service
-	// TODO [multi-tenancy] replace this with config service
-	config config.Configuration
+	config  configstore.Service
 }
 
 // GRPCHandler returns an implementation of the purchaseorder DocumentServiceServer
-func GRPCHandler(config config.Configuration, registry *documents.ServiceRegistry) (clientpurchaseorderpb.DocumentServiceServer, error) {
+func GRPCHandler(config configstore.Service, registry *documents.ServiceRegistry) (clientpurchaseorderpb.DocumentServiceServer, error) {
```

Comment:
We should start thinking on renaming this 😄

Reply:
oh I realise this is not a p2p handler so I believe it still makes sense to keep this until we remove grpc from API.

Reply:
yep, not asking to do it now.